lib/test_suite.py

```diff
@@ -174,6 +174,14 @@ def fragile_tests(self):
                 res.append(test)
         return res
 
+    def fragile_checksums(self):
```

Comment:
Maybe fragile_checksums()  should return a dict {test_name_1 -> [checksum1, checksum2],
test_name_2 -> [checksum1]}

Reply:
Sure, added ability to use JSON format.